packages/providers/src/test-contract/src/main.sw

```diff
@@ -8,29 +8,29 @@ struct TestStruct {
 }
 
 abi TestContract {
-  fn foo(gas_: u64, amount_: u64, coin_: b256, value: u64) -> u64;
-  fn boo(gas_: u64, amount_: u64, coin_: b256, value: TestStruct) -> TestStruct;
-  fn barfoo(gas_: u64, amount_: u64, coin_: b256, value: u64) -> u64;
-  fn foobar(gas_: u64, amount_: u64, coin_: b256, value: ()) -> u64;
+  fn foo(value: u64) -> u64;
+  fn boo(value: TestStruct) -> TestStruct;
+  fn barfoo(value: u64) -> u64;
+  fn foobar(value: ()) -> u64;
 }
 
 impl TestContract for Contract {
-  fn foo(gas_: u64, amount_: u64, asset_id_: b256, value: u64) -> u64 {
+  fn foo(value: u64) -> u64 {
     log_u64(3735928559);
     value
   }
-  fn boo(gas_: u64, amount_: u64, asset_id_: b256, value: TestStruct) -> TestStruct {
+  fn boo(value: TestStruct) -> TestStruct {
```

Comment:
Technically, we can break this struct into multiple arguments now but I suppose it's good to keep a struct test. That being said, maybe it's a good idea to have another method here that requires more than 1 argument?

Reply:
I will not address this in this PR.